app.py

```diff
@@ -1,9 +1,13 @@
+import hashlib
+
 import streamlit as st
 
+from flower_classifier.datasets.flickr.client import get_authenticated_client, upload_photo
 from flower_classifier.datasets.oxford_flowers import NAMES as oxford_idx_to_names
 from flower_classifier.ui import components
 
 WEIGHTS_URL = "https://github.com/jeremyjordan/flower-classifier/releases/download/v0.1/efficientnet_b3a_example.ckpt"
+SUCCESSFUL_UPLOADS = set()
```

Comment:
Where is this used?

Reply:
Oops, this is no longer being used. Good catch!